displayFunction.php

```diff
@@ -20,17 +20,31 @@ function createTile ($book): string
         . hideIFNull($book->getRating(), '<p class="book-detail" >Rating: ' . $book->getRating() . '/10</p>')
         . hideIfNull($book->getGrLink(),
             '<a href="'.$book->getGrLink()
-            .' "target="_blank" class="book-detail"><i class="fa-brands fa-goodreads"></i></a>')
+            .'" target="_blank" class="book-detail"><i class="fa-brands fa-goodreads"></i></a>')
         . '</div>'
         . '</div>';
 }
 
 /**
- * @param $var - Book object
+ * @param $obj - Book object
  * @param $string - HTML string
  * @return string
  */
 function hideIfNull($obj, $string): string
 {
     return ($obj == null) ? '<p class="book-detail"></p>' : $string;
-}
\ No newline at end of file
+}
+
+/**
+ *
+ */
+function sanitiseUrl($url): string
+{
+    $url = filter_var($url, FILTER_SANITIZE_URL);
+
+    if (filter_var($url, FILTER_VALIDATE_URL) !== false) {
+        return $url;
+    } else {
+        return "";
+    }
```

Comment:
You can ditch the else here! 

Have a think about what I might mean, then give me a shout!